core/src/test/java/nl/nn/adapterframework/filesystem/FileSystemListenerTest.java

```diff
@@ -191,6 +191,34 @@ public void fileListenerTestGetRawMessageWithInProcess() throws Exception {
 		fileListenerTestGetRawMessage(null,folderName);
 	}
 
+	@Test
+	public void fileListenerTestGetRawMessageWithInProcessTimeSensitive() throws Exception {
```

Comment:
Could you also make tests for 
- handling of two files with the same name but different timestamps
- handling of two files with the same name and with the same timestamp